src/ResizableBox.tsx

```diff
@@ -106,16 +107,26 @@ const ResizeHandle: React.FC<{
 
 export const ResizableBox: React.FC<Omit<BoxProps, 'width'>> = ({
   children,
+  hidden,
   ...props
 }) => {
+  const { isEmbedded } = useEmbed();
```

Comment:
`ResizableBox` sounds like a quite generic component, decoupled from our business logic - this makes me think that reading this state here should not be its responsibility. It could accept a `disabled` prop or something instead of that

Reply:
I agree. fixing